packages/calcite-components/src/utils/dynamicCSSConst.ts

```diff
@@ -0,0 +1,7 @@
+export const getWidth = (width: string, widthScale: string): string => {
```

Comment:
Since `height` and `width` are shared, WDYT about adding a `CSS` export with those classes for other components to use?

Reply:
As per our convo, because we now use props to generate the scale (and the util is applied throughout) I'm not sure where we'd have a use for `CSS.width` or `CSS.height`.  @jcfranco